src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.OSX.cs

```diff
@@ -179,6 +179,7 @@ private static unsafe void CreateAndStartRunLoop()
             }
             s_runLoopThread = new Thread(RunLoopThreadStart);
             s_runLoopThread.IsBackground = true;
+            s_runLoopThread.Name = ".NET Net Address Change";
```

Comment:
Address Change => Monitor?
Net => Network?
e.g. ".NET Network Monitor" or ".NET Net Address Monitor"?

Reply:
I contracted Network. I'll use ".NET Net Address Monitor" as it contracts usefully.